DataSet strict mode, UMS Namespaces, Policy Filters, Delete project (#28)
jexia/docs: docs/get-started/README.md

## Delete project

> To delete your project you need to first remove the application from within the **Application Hosting** section. Then go to **Settings** and click **Delete**, with this email with one time token will be sent to your email. Please use this token to confirm project deletion. 

Reviewer: ```suggestion
To delete your project you need to first remove the application from within the **Application Hosting** section. Then go to **Settings** and click **Delete**, a one time token will be sent to the accounts email. You will need to use this token to confirm project deletion. 
```
